gcs/CHANGES.md

```diff
@@ -35,42 +35,49 @@
     fs.gs.cooperative.locking.max.concurrent.operations
     ```
 
-1.  Migrate authentication to `com.google.auth.Credentials` and remove obsolete
-    properties:
+10. Migrate authentication to `com.google.auth.Credentials` and remove obsolete
+     properties:
 
-    ```properties
-    fs.gs.auth.service.account.email
-    fs.gs.auth.service.account.keyfile
-    fs.gs.auth.service.account.private.key
-    fs.gs.auth.service.account.private.key.id
-    ```
+     ```properties
+     fs.gs.auth.service.account.email
+     fs.gs.auth.service.account.keyfile
+     fs.gs.auth.service.account.private.key
+     fs.gs.auth.service.account.private.key.id
+     ```
 
-1.  Refactor authentication configuration to use an explicit `fs.gs.auth.type`
-    enum property, instead of relying on inference of the authentication type
-    based on the set configuration properties, and remove obsolete properties:
+11. Refactor authentication configuration to use an explicit `fs.gs.auth.type`
+     enum property, instead of relying on inference of the authentication type
+     based on the set configuration properties, and remove obsolete properties:
 
-    ```properties
-    fs.gs.auth.null.enable
-    fs.gs.auth.service.account.enable
-    ```
+     ```properties
+     fs.gs.auth.null.enable
+     fs.gs.auth.service.account.enable
+     ```
 
-1.  Add support for a new `USER_CREDENTIALS` authentication type that retrieves
-    a refresh token using the authorisation code grant flow configured via the
-    following properties:
+12. Add support for a new `USER_CREDENTIALS` authentication type that retrieves
+     a refresh token using the authorisation code grant flow configured via the
+     following properties:
 
-    ```properties
-    fs.gs.auth.client.id
-    fs.gs.auth.client.secret
-    fs.gs.auth.refresh.token
-    ```
+     ```properties
+     fs.gs.auth.client.id
+     fs.gs.auth.client.secret
+     fs.gs.auth.refresh.token
+     ```
 
-1.  Merge all output stream types functionality in the default output stream
-    that behaves similarly to the `FLUSHABLE_COMPOSITE` stream, and remove
-    obsolete property:
+13. Merge all output stream types functionality in the default output stream
+     that behaves similarly to the `FLUSHABLE_COMPOSITE` stream, and remove
+     obsolete property:
+
+     ```properties
+     fs.gs.outputstream.type
+     ```
+
+14. Updated default value for max item to return from list Cloud Storage
+    requests via following property:
 
     ```properties
-    fs.gs.outputstream.type
-    ```
+     fs.gs.list.max.items.per.call
+     ```
```

Comment:
We usually document default values changes like:

```md
1.  Set default value for `fs.gs.list.max.items.per.call` property to `5000`.
```

Reply:
Reverted.